client/src/main/com/sinch/sdk/domains/voice/adapters/converters/DestinationDtoConverter.java

```diff
@@ -2,6 +2,7 @@
 
 import com.sinch.sdk.domains.voice.models.Destination;
 import com.sinch.sdk.domains.voice.models.DestinationNumber;
+import com.sinch.sdk.domains.voice.models.DestinationNumber.TYPE;
```

Comment:
?

Reply:
This is breaking existing pattern for repo, thank you
Fixed by creating `DestinationNumberType`